bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -26,8 +27,14 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList', function (id) {
      * @param {Object} options
      * @return {jQuery Element} a list with chosen applied
      */
-    create: function (data, options) {
-        options = options || this.defaultOptions;
+    create: function (data, modifiedOptions) {
+        var me = this;
+        var getOptions = function (options) {
+            // extend default options to create new object where we concat passed values
+            return jQuery.extend(true, {}, me.defaultOptions, options);
+        };
+        var options = getOptions(modifiedOptions);
```

Comment:
I can't see why getOptions should be a function that does nothing, but calls another function.